windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -397,12 +434,12 @@ procedure TMergeKeyboardInfo.AddName;
 
   if Assigned(FKMPInfFile) then
     FName := FKMPInfFile.Info.Desc[PackageInfo_Name]
-  else if Length(FJSFiles) > 0  then
+  else if (FJSFileInfo.Data <> '')  then
```

Comment:
Extra parentheses not required; use `if not FJSFileInfo.Data.IsEmpty then` which is more fluent.